tests/unit/amo/reducers/test_versions.js

```diff
@@ -235,27 +235,6 @@ describe(__filename, () => {
       ).toEqual(null);
     });
 
-    it('returns null for created and filesize when no file is found', () => {
-      const version = {
-        ...fakeVersion,
-        files: [],
-      };
-      const state = versionsReducer(
-        stateWithLang,
-        loadVersions({
-          slug: 'some-slug',
-          versions: [version],
-        }),
-      );
-
-      expect(
-        _getVersionInfo({
-          state,
-          versionId: fakeVersion.id,
-        }),
-      ).toMatchObject({ created: null, filesize: null });
-    });
```

Comment:
same here

Reply:
where is the code that would be tested here, then?